Lib/ipaddress.py

```diff
@@ -469,9 +469,12 @@ def _prefix_from_ip_int(cls, ip_int):
         return prefixlen
 
     @classmethod
-    def _report_invalid_netmask(cls, netmask_str):
+    def _report_invalid_netmask(cls, netmask_str, suppress=True):
         msg = '%r is not a valid netmask' % netmask_str
-        raise NetmaskValueError(msg) from None
+        if suppress:
```

Comment:
In some cases there is no other exception to suppress. When we call this function not from the `except` block.

Reply:
You can check whether ``sys.exception()`` is ``None``.

Reply:
Though, do you need to? What's the harm in "from None" when you are not in an except block?